src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -4175,6 +4175,22 @@ namespace winrt::TerminalApp::implementation
         const auto theme = _settings.GlobalSettings().CurrentTheme();
         auto requestedTheme{ theme.RequestedTheme() };
 
+        {
+            // Update the brushes that Pane's use...
+            Pane::SetupResources(requestedTheme);
+            // ... then trigger a visual update for all the bane borders to
```

Comment:
```suggestion
            // ... then trigger a visual update for all the pane borders to
```
![BaneBatmanGIF](https://user-images.githubusercontent.com/11050425/205740668-ecc6f3b8-c221-452e-9b60-831aef09c3ad.gif)